packages/x-components/tests/e2e/cucumber/url/url.feature

```diff
@@ -6,19 +6,20 @@ Feature: Url component
     And   a next queries API
     And   a suggestions API
     And   a related tags API
+    And   a tracking API
 
   Scenario Outline: 1. Navigating to a URL from the outside sets the url origin as "<origin>"
     Given a URL with query parameter "lego"
-    And   waiting for search request intercept with new origin
     Then  the search request contains the origin "<origin>"
     Examples:
-      | origin        |
-      | url:external  |
+      | origin       |
+      | url:external |
 
   Scenario Outline: 2. Navigate back and forth in serp sets the url origin as "<origin>"
     Given a URL with query parameter "lego"
+    Then  the search request contains the origin "url:external"
```

Comment:
Should we use a variable for the first origin too?

Reply:
I wouldn't. I think it is a lot more confusing to see things like `firstOrigin`, `secondOrigin`, rather than the hardcoded value. I don't even like to have tables if tests only have 1 example 🤣.